core/src/test/scala/org/apache/spark/CheckpointSuite.scala

```diff
@@ -37,40 +41,42 @@ class CheckpointSuite extends SparkFunSuite with LocalSparkContext with Logging
     sc.setCheckpointDir(checkpointDir.toString)
   }
 
-  override def afterEach() {
+  override def afterEach(): Unit = {
     super.afterEach()
     Utils.deleteRecursively(checkpointDir)
   }
 
-  test("basic checkpointing") {
+  runTest("basic checkpointing") { normal: Boolean =>
```

Comment:
What does `normal` mean!? Please rename it to something like `isLocalCheckpoint`